templates/swift/Sources/OAuth/WebAuthComponent.swift.twig

```diff
@@ -92,7 +103,7 @@ public class WebAuthComponent {
             cookie += "; secure"
         }
 
-        let existing = UserDefaults.standard.stringArray(forKey: domain)
+        _ = UserDefaults.standard.stringArray(forKey: domain)
```

Comment:
Not sure we even need this at all if the read result can be discarded 🤔

Reply:
I was also not aware of the purpose, this just silenced the obvious compiler warning.  I wonder if it was part of a previous attempt to set the cookie for the domain.

Reply:
Yep probably my own leftover 😅